config/config.go

```diff
@@ -213,6 +210,9 @@ func Marshal(p *pb.ServiceConfig) (io.Reader, error) {
 }
 
 func Unmarshal(r io.Reader) (*pb.ServiceConfig, error) {
+	if r == nil {
+		panic("R is NIL")
```

Comment:
s/R/reader?

Reply:
LOL - this was temporary debugging code. It should return an error, not panic. Thanks for the catch.